lib/controllers/v1/observations_controller.js

```diff
@@ -768,28 +768,41 @@ ObservationsController.reqToElasticQueryComponents = function( req, callback ) {
 ObservationsController.resultsForRequest = function( req, callback ) {
   ObservationsController.elasticResults( req, function( err, data ) {
     if( err ) { return callback( err ); }
-    var obs = _.map( data.hits.hits, function( h ) {
-      return new Observation( h._source, { userSession: req.userSession } );
-    });
-    if( err ) { return callback( err ); }
-    var response = { total_results: data.hits.total };
-    if (
-      data.aggregations &&
-      data.aggregations.bbox &&
-      data.aggregations.bbox.bounds &&
-      data.aggregations.bbox.bounds.bottom_right
-    ) {
-      response.total_bounds = {
-        swlat: data.aggregations.bbox.bounds.bottom_right.lat,
-        swlng: data.aggregations.bbox.bounds.top_left.lon,
-        nelat: data.aggregations.bbox.bounds.top_left.lat,
-        nelng: data.aggregations.bbox.bounds.bottom_right.lon
-      };
+    function handleObs( obs ) {
+      obs = _.map( obs, function( o ) {
+        return new Observation( o, { userSession: req.userSession } );
+      });
+      if( err ) { return callback( err ); }
+      var response = { total_results: data.hits.total };
+      if (
+        data.aggregations &&
+        data.aggregations.bbox &&
+        data.aggregations.bbox.bounds &&
+        data.aggregations.bbox.bounds.bottom_right
+      ) {
+        response.total_bounds = {
+          swlat: data.aggregations.bbox.bounds.bottom_right.lat,
+          swlng: data.aggregations.bbox.bounds.top_left.lon,
+          nelat: data.aggregations.bbox.bounds.top_left.lat,
+          nelng: data.aggregations.bbox.bounds.bottom_right.lon
+        };
+      }
+      response.page = Number( req.elastic_query.page );
+      response.per_page = Number( req.elastic_query.per_page );
+      response.results = obs;
+      callback( null, response );
+    }
+    // If this is an authenticated request or all details request, preload
+    // project observation preferences so we can show/hide coordinates correctly
+    var obs = data.hits.hits.map( h => h._source );
+    if ( req.userSession || req.query.details === "all" ) {
```

Comment:
I understand this is necessary to get the private coords when not using details == "all", but I worry about performance. Loading the project observation preferences is nearly single handedly the culprit for slow loads on observations show (it's a very slow DB query). This won't be a problem after the data is added to the index. 

We don't need to change it now, but once this is released I'm going to keep my eye out for slow loading observation searches on the web and apps. If performance becomes a problem I'd probably advocate only doing this only when details === "all", until the index is updated (likely next week)